src/leveleditor/leveleditor.cpp

```diff
@@ -397,7 +397,8 @@ int main(int argc, char *argv[])
 	printf("\n---------------- startup ----------------\n");
 
     {
-        BinaryFile editor_settings("leveleditor.bin", "rb");
+        std::string options_path(GetHomeDirectory() + "leveleditor.bin");
```

Comment:
If `options_path` won't get modified later on, we could mark it as `const`.

Reply:
Done.